src/Domain/Ci/Job/IntegratedTestOnLatestLtsCiJob.php

```diff
@@ -34,8 +50,10 @@ class IntegratedTestOnLatestLtsCiJob extends AbstractCiJob {
    * @param \Acquia\Orca\Helper\Process\ProcessRunner $process_runner
    *   The process runner.
    */
-  public function __construct(EnvFacade $env_facade, ProcessRunner $process_runner) {
+  public function __construct(DrupalCoreVersionResolver $drupal_core_version_resolver, EnvFacade $env_facade, OutputInterface $output, ProcessRunner $process_runner) {
```

Comment:
Seems like you alphabetize your arguments, I tried to be consistent 😄

Reply:
That's what makes you a rockstar, @danepowell. 😄